Connecting to a cluster in AWS (#2532)
EnterpriseDB/docs: product_docs/docs/biganimal/release/overview/03_security/index.mdx

-   **Granular access control:** You can use single sign-on (SSO) and define your own sets of roles and role-based access control (RBAC) policies to manage your individual cloud environments. See [Managing portal access](../administering_cluster/01_portal_access) for more information. 

> -   **Data encryption:** All data in BigAnimal is encrypted in motion and at rest. Network traffic is encrypted using Transport Layer Security (TLS) v1.2 or greater, where applicable. Data at rest is encrypted using AES with 256-bit keys. Data encryption keys are envelope encrypted, and the wrapped data encryption keys are securely stored by your cloud provider in your account. Encryption keys never leave your environment.

Reviewer: ```suggestion
-   **Data encryption:** All data in BigAnimal is encrypted in motion and at rest. Network traffic is encrypted using Transport Layer Security (TLS) v1.2 or greater, where applicable. Data at rest is encrypted using AES with 256-bit keys. Data encryption keys are envelope encrypted, and the wrapped data encryption keys are securely stored in a key management system in your account. Encryption keys never leave your environment.
```

Author: Fixed.
